test/get_test.go

```diff
@@ -13,13 +12,11 @@ import (
 func TestGet(t *testing.T) {
 	ctx := context.Background()
 	client := newKine(t)
-
-	i := 0
+	var key string
 
 	t.Run("FailNotFound", func(t *testing.T) {
 		g := NewWithT(t)
-		key := fmt.Sprintf("testKey-%d", i)
-		i++
+		key = "testKeyFailNotFound"
```

Comment:
key can just be local on each test, eg
```suggestion
		key := "testKeyFailNotFound"
```